app/src/pages/inside/stepPage/stepGrid/defectType/autoAnalyzedLabel/autoAnalyzedLabel.scss

```diff
@@ -17,7 +17,7 @@
 .aa-label {
   display: inline-block;
   height: 20px;
-  min-width: 43px;
+  width: 43px;
```

Comment:
Why are you reverting your previous changes?

Reply:
It looks ugly in firefox browser